tlsobs-api/handlers.go

```diff
@@ -439,6 +439,38 @@ func TruststoreHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// IssuerStatusHandler handles the /issuerStatus endpoint of the api.
+// It queries the database for a list of end-entity cert_ids which chain via the
+// given certificate.
+// It takes the following HTTP parameter:
+//     fingerprint - a base64 encoded sha256 certificate fingerprint
+func IssuerStatusHandler(w http.ResponseWriter, r *http.Request) {
+	val := r.Context().Value(ctxDBKey)
+	if val == nil {
+		httpError(w, r, http.StatusInternalServerError,
+			fmt.Sprintf("Could not find database handler in request context"))
+		return
+	}
+	db := val.(*pg.DB)
+	certIDs, err := db.GetAllCertIDsIssuedBy(r.FormValue("fingerprint"));
```

Comment:
On other endpoints, we use `sha256` instead of fingerprint. I'd prefer to stay consistent.